tests/test_printallocations.py

```diff
@@ -0,0 +1,116 @@
+
+import unittest
+import sys
+
+from models.dojo import Dojo
+
+
+class TestPrintAllocation(unittest.TestCase):
+    """It tests for the functionality print room."""
+
+    allocation_office = "green"\
```

Comment:
If you were trying to get a multiline string, python has the tripple quotes
"""

"""